### DISCUSSION TITLE
Establishing core idea and documentation

### DISCUSSION THREAD

**Discussion by Rutuj-Runwal** [Soham-Mundhada/CareerRecommenderEngine#3] [Q&A]
The idea of creating a unique career platform is great although, the repository dosent have basic course of action and description.
I would like to kick start this Hacktoberfest by contributing to creation of this recommender engine.

Can I start with creating this @Soham-Mundhada

**Comment by Soham-Mundhada** [comment]
> The idea of creating a unique career platform is great although, the repository dosent have basic course of action and description. I would like to kick start this Hacktoberfest by contributing to creation a this recommender engine.
> 
> Can I start with creating this @Soham-Mundhada

Hi Rutuj.
Sure, I will be glad to add you as a contributor on this project and work together. 
Welcome aboard.
